exonum/src/encoding/protobuf/mod.rs

```diff
@@ -216,6 +216,16 @@ impl ProtobufConvert for u64 {
     }
 }
 
+impl ProtobufConvert for i32 {
+    type ProtoStruct = i32;
+    fn to_pb(&self) -> Self::ProtoStruct {
+        *self
+    }
+    fn from_pb(pb: Self::ProtoStruct) -> Result<Self, ()> {
+        Ok(pb)
+    }
+}
```

Comment:
I think it makes sense now to just add

```rust
macro_rules! trivial_protobuf_type {
    ($T:ty) => {
        impl ProtobufConvert for $T {
            type ProtoStruct = $T;
            fn to_pb(&self) -> Self::ProtoStruct {
                *self
            }
            fn from_pb(pb: Self::ProtoStruct) -> Result<Self, ()> {
                Ok(pb)
            }
        }
    }
}
```

already and implement `ProtobufConvert` for all standard integer types.

Reply:
Yes, that was discussed already. I agree.

Reply:
@dvush add this in separate PR ?